src/main/java/blackjack/domain/BlackjackGame.java

```diff
@@ -110,18 +113,35 @@ public int getPlayerScore(String name) {
         return participants.findByName(name).getScore();
     }
 
-    public GameResultDto getDelearGameResultDto() {
-        Person dealer = participants.getDealer();
-        List<GameResult> dealerGameResults = participants.getPlayers()
-                .stream()
-                .map(dealer::matchGame)
-                .collect(toList());
-        return GameResultDto.of(dealerGameResults);
+    public Map<String,Double> getParticipantsProfit(Map<String, Integer> bettingMoney) {
+        Exchanger exchanger = new Exchanger(bettingMoney);
+        Map<String, Double> playersProfit = getPlayersProfit(exchanger);
+        double dealerProfit = getDealerProfit(exchanger, playersProfit);
+
+        Map<String, Double> participantsProfit = new HashMap<>(playersProfit);
+        participantsProfit.put(participants.getDealer().getName(), dealerProfit);
```

Comment:
```suggestion
        participantsProfit.put(participants.getDealerName(), dealerProfit);
```

getter를 두 번 가져오기 보단 객체 안에서 바로 필요한 값을 넘겨줄 수 있게 하면 어떨까요? 다른 코드에서도 이런 부분들이 보여서 같이 리팩터링 해보면 좋을 듯 싶어요!

Reply:
수정했습니다!